stock_picking_restrict_cancel_printed/tests/test_stock_picking_restrict_cancel_printed.py

```diff
@@ -65,3 +65,13 @@ def test_stock_move_restrict_cancel_printed_enabled_nobackorder(self):
         self.picking.move_ids.quantity_done = 1
         self.picking_type.create_backorder = "never"
         self.picking.button_validate()
+
+    def test_stock_move_restrict_cancel_printed_enabled_move_merge(self):
+        """Check a picking processed in excess can be validated
+
+        This will trigger the creation of an extra move.
+        That extra gets merged and then canceled.
+        """
+        self.picking.printed = True
```

Comment:
Shouldn't be `restrict_cancel_if_printed` set to true.
Otherwise it will always call super._action_cancel?
```suggestion
        self.picking_type.restrict_cancel_if_printed = True 
        self.picking.printed = True
```

Reply:
there is a default=True, so should be the case